beginner_source/blitz/tensor_tutorial.py

```diff
@@ -51,6 +52,16 @@
 
 
 ######################################################################
+# **Tensor as Diagonal Matrix
+#
+# Like NumPy operations, the eye function creates a diagonal matrix of which the diagonal elements have ones and 
+# off diagonal elements have zeros
+
+x_eye = torch.eye(3, 4)
```

Comment:
Why the `eye` operator is interesting here? What does it adds to the tutorial?

Reply:
The eye operator is an interesting operator for tensors that shows there are operations defined in tensors also just like numpy with which one can create a diagonal matrix easily with a one line code.